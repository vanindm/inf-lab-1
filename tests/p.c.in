5 4
0.03257,0.09946 0.0946,0.09268 0.07386,0.06674 0.02811,0.02592 
0.0555,0.0919 0.03977,0.06473 0.0544,0.06527 0.06858,0.06221 
0.05155,0.06453 0.009,0.03678 0.0273,0.02254 0.09825,0.03286 
0.08363,0.03246 0.00342,0.05849 0.04758,0.00194 0.06827,0.00345 
0.0129,0.0825 0.09659,0.02304 0.01915,0.02179 0.08452,0.01401 
4 5
0.02354,0.03679 0.05153,0.00769 0.07798,0.00808 0.06202,0.09312 0.02549,0.0375 
0.07172,0.02318 0.09597,0.05715 0.01421,0.02446 0.09468,0.07623 0.0263,0.03542 
0.08002,0.05326 0.00986,0.09238 0.00209,0.05597 0.01271,0.0904 0.05739,0.05307 
0.06549,0.05019 0.08195,0.02818 0.04381,0.09795 0.03289,0.0084 0.025,0.01347 
