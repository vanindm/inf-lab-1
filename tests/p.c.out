0.004640,0.024762 0.000832,0.030074 -0.004075,0.019812 -0.009738,0.033802 -0.002644,0.018321 
0.001523,0.025406 0.000644,0.026347 -0.004061,0.022131 -0.009761,0.028398 -0.002744,0.016647 
0.004402,0.016603 0.006234,0.015972 0.002608,0.018830 -0.003499,0.017631 0.000212,0.009751 
0.007666,0.014457 0.006833,0.014746 0.007521,0.013627 0.000675,0.020612 0.003221,0.009265 
0.008866,0.014231 0.012691,0.017596 0.002299,0.019237 0.001442,0.019041 0.000826,0.010369 
